app/grandchallenge/core/static/css/core.css

```diff
@@ -168,3 +171,20 @@ ul.socialaccount_providers > li {
 .phaseButton:focus, .phaseButton:hover {
     box-shadow:none !important;
 }
+
+.card, .card-header {
+    border-radius: 20px;
+}
+
+.card:hover {
+    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
```

Comment:
Just curious, why did you use a double shadow here?

Styling nitpick: I like box-shadow without x/y offset making the shadow drop evenly around the element. Example:
Current: 
![image](https://user-images.githubusercontent.com/2001094/135245022-bbe3cb79-f9a4-444d-95ba-1dad85534705.png)

Even:
![image](https://user-images.githubusercontent.com/2001094/135245207-740fc6c3-80b2-4081-a76f-e143dc50e366.png)

(Personal preference, do with it what you want)